recipe/meta.yaml

```
@@ -7,46 +7,52 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytorch_tabnet-{{ version }}.tar.gz
-  sha256: 18887b993a8bf86ec05a576b5cf93f09e08b778cd9f418c5b254b6566df673a5
+  url: https://github.com/dreamquark-ai/tabnet/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 0703d0298bf605dae289515023859428017fe86e4663de9068a99f06b9802c94
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-build-isolation --no-deps
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - wheel
+    - python
     - poetry >=0.12
```

```suggestion
    - poetry-core >=0.12
```
That's all that is actually needed to build.

---

I actually tried that, but it failed: `ModuleNotFoundError: No module named 'poetry.masonry'`

---

Interesting, apparently changing to poetry-core, the backend should be updated to poetry.core.masonry.api in pyproject.toml.
But nevermind as it works with poetry.